ImmuDB4Net/ImmuClient.cs

```diff
@@ -337,6 +483,10 @@ public async Task UseDatabase(string database)
         currentDb = database;
     }
 
+    /// <summary>
+    /// Gets the server's database list
+    /// </summary>
```

Comment:
Please add a remark that this will only return databases the user has access to.

Reply:
ok